hack/dependencies.yml

```diff
@@ -73,11 +73,11 @@
   version: v1.1.1
 - checksums:
     linux:
-      amd64: 5e7ecb614b5926acfc36eb1258800391ab7c6e6e026fa7cacbfe92006bac895c
-      arm64: a8c3f4140d18c324cc69f5de4df0566e529e1636cff340095a42475799bf3fed
+      amd64: 38c9a2f484076aeafd9f522efdee40538c31337539bd8c80a29f5c4077314e53
+      arm64: 735fa1b9bb02ef0ee79dd40c418760687776b44747f43f2e26c3bca4e1fd96f6
   dev: false
   name: cue
   repo: cue-lang/cue
   tarballSubpath: cue
   urlTemplate: https://github.com/cue-lang/cue/releases/download/{{.Version}}/cue_{{.Version}}_{{.OS}}_{{.Arch}}.tar.gz
-  version: v0.4.3
+  version: v0.5.0
```

Comment:
Do we bump minors for patch releases? (I am not sure if we decide something on this)

Reply:
I'm also not sure about this. I have just bumped it as it's been a month since it was released.

Reply:
Reverted the changes.